### PR TITLE
Template releng ticket for upstream binary signing

### PR DESCRIPTION
**Pull request by bgilbert**
The releng signing tickets for Butane and Ignition are currently created by manually finding an old ticket, modifying it, copying the embedded shell script to a file, running it, and seeing whether it works.  To avoid all these manual steps, add a templated shell script that generates the ticket contents, and that also knows how to directly extract and run the embedded shell script for testing.  The result is a template containing a shell script containing Markdown containing a shell script, but in my defense (?), I also enjoy Autotools.

While we're here, add [Butane](https://src.fedoraproject.org/rpms/butane/pull-request/11) and [Ignition](https://src.fedoraproject.org/rpms/ignition/pull-request/93) aarch64 macOS binaries.

### CODE REVIEW

**Comment by bgilbert** [comment]
https://github.com/coreos/fedora-coreos-tracker/pull/1242 adds a FCOS release checklist step to update the Fedora version and signing key in `config.yaml`.

**Review by travier — APPROVED** [review]
Quick/weak LGTM

File: go/release-checklist.md [hunk 1]
```diff
@@ -47,15 +47,14 @@ Fedora packaging:
 
 GitHub release:
  - [ ] Wait until the Bodhi update shows "Signed :heavy_check_mark:" in the Metadata box.
- - [ ] [File a releng ticket](https://pagure.io/releng/new_issue) based on [prior signing tickets]({{ sample_signing_ticket }}).
-   - [ ] Update the script and test it locally by running it like `FAKESIGN=1 ./script`
-     - [ ] If a new Fedora release has gone stable, update the signing key in the script to use the new Fedora signing key [found here](https://getfedora.org/security).
+ - [ ] Run `bash signing-ticket.sh ticket <x.y.z>` and paste the output into a [releng ticket](https://pagure.io/releng/new_issue).
+   - [ ] Test the embedded script locally by running `bash signing-ticket.sh test <x.y.z> <output-dir>`
```
[the remark this answers is not in the document]
Comment (bgilbert) [reply go/release-checklist.md]: Yeah, I had kept the existing ambiguous ordering.  Fixed.

**Comment by bgilbert** [comment]
Updated.  I also decided the script shouldn't assume that the release binaries are in `Release` 1 of the package, so I've parameterized that.

**Review by jlebon — APPROVED** [review]

**Pull request merged**